src/components/Info/Info.jsx

```diff
@@ -15,36 +16,54 @@ function Info(props) {
         setActiveTab(2)
     }
 
+    // TO-DO: refactor this code to improve how we're handling text
     return <Modal>
-        <ModalContent>
             <TabBar fixed>
                 <Tab onClick={handleTabSwitch1} selected={activeTab === 1} dataTest="dhis2-uicore-tab">{i18n.t("User Guide")}</Tab>
                 <Tab onClick={handleTabSwitch2} selected={activeTab === 2} dataTest="dhis2-uicore-tab">{i18n.t("Additional Features")}</Tab>
             </TabBar>
+            <ModalContent className={styles.info}>
             {activeTab === 1 ? <div>
-                <h5>How to reate new catchment areas</h5>
-                <p>Click INSERT BUTTON</p>
-                <p>Select the country you want to target</p>
-                <p>Name the catchment areas</p>
-                <p>Choose the admin level where your facilities are located</p>
-                <p>Choose which groups of facilities to include</p>
-                <p>You can use your catchment areas to create thematic maps and other views in the maps module of DHIS2</p>
-                <p>1</p>
+                <ModalTitle>{i18n.t("How to create new catchment areas")}</ModalTitle>
+                <p className={styles.align}>{i18n.t("Click")}<ButtonItem primary={true} buttonText={i18n.t("Create")}/></p>
```

Comment:
I don't think this button should be clickable, unless we want it to actually cause the Create dialog.

Reply:
The button does not have any actions associated with it. If the user clicks on it, it shouldn't prompt anything. There is a hover action, it might be confusing. Any suggestions?

Reply:
Is there a `disable` option for the button?

Reply:
There is but it's visibly different.